app/controllers/tt/api/v1/visitor_details_controller.rb

```diff
@@ -0,0 +1,5 @@
+class Tt::Api::V1::VisitorDetailsController < Tt::Api::V1::ApplicationController
+  def show
+    render json: TestTrack::FakeServer.visitor_details
```

Comment:
Yea, I _think_ this is fine. We really just want to make this fakes as lifelike as they need to be in order to work. They can be made more and more lifelike as we have a stronger need for that.

So, I think this is fine.